pacta/README.md

```diff
@@ -1,91 +1,122 @@
-- remove any existing "2dii_pacta" images from your local docker system
-- build a "2dii_pacta:<tag>" and "2dii_pacta:latest" docker image (where <tag>
-is the tag you provided, e.g. 0.0.4). The image builds from the Dockerfile
-in this directory, which will
-  - use 2dii/r-packages as a base
-  - copy in the freshly cloned repos
-  - make some necessary permissions changes
-  - export the freshly made docker image gzipped (2dii_pacta.tar.gz)
+-   clone the repos locally, only copying the current version of the
+    files
+
+-   build a "2dii_pacta:<tag>" and "2dii_pacta:latest" docker image
+    (where <tag> is the tag you provided, e.g. 0.0.4). The image builds
+    from the Dockerfile in this directory, which will
+
+    -   use 2dii/r-packages as a base
+    -   copy in the freshly cloned repos
+    -   make some necessary permissions changes
+    -   export the freshly made docker image gzipped (2dii_pacta.tar.gz)
 
-If the build is successful, load the image interactively and push the tags
-created for each of the PACTA_analysis and friends repos inside of the
-docker container.
+If the build is successful, load the image interactively and push the
+tags created for each of the PACTA_analysis and friends repos inside of
+the docker container.
 
 # Releasing
 
-To release a new version of the software, push all tags to GitHub. The tag you provide will affect the image, the repository containing the dockerfile that builds the image, and the pacta siblings. That consistent tag make the process reproducible.
+To release a new version of the software, push all tags to GitHub. The
+tag you provide will affect the image, the repository containing the
+dockerfile that builds the image, and the pacta siblings. That
+consistent tag make the process reproducible.
 
 To push the tags, start the container with something like
 
-```bash
+``` {.bash}
 cd ~
 docker run --rm -ti -v "$(pwd)/.ssh":/root/.ssh 2dii_pacta:latest
 ```
 
-This example starts an ephemeral container (`run --rm`) from the image `2dii_pacta:latest`, and creates a volume that makes your .ssh key available to the container (`-v "$(pwd)/.ssh":/bound/.ssh`), which you'll need to interact with GitHub. If instead of ssh you use https protocol, you may omit the volume argument and provide your username and password when prompted.
+This example starts an ephemeral container (`run --rm`) from the image
+`2dii_pacta:latest`, and creates a volume that makes your .ssh key
+available to the container (`-v "$(pwd)/.ssh":/bound/.ssh`), which
+you'll need to interact with GitHub. If instead of ssh you use https
+protocol, you may omit the volume argument and provide your username and
+password when prompted.
 
-Once inside the container, you can push any tag as you would normally push a branch `git push <remote> <tag>`, for example:
+Once inside the container, you can push any tag as you would normally
+push a branch `git push <remote> <tag>`, for example:
 
-```bash
+``` {.bash}
```

Comment:
I'm curious: What does this change do?

Reply:
RStudio did that on it's own... I presume it's a more "proper" way of defining the type of source code being displayed (!?!)...

https://pandoc.org/MANUAL.html#fenced-code-blocks
https://github.com/rstudio/rmarkdown/issues/1073#issuecomment-306924809

Reply:
Thanks! 

I don't know what I prefer:

I like the syntax `{.language}` because I can make the chunk executable simply by deleting the `.`. But I dislike it because it's harder to predict that the chunk will run simply by skimming the code. 

:)